.NET/Microsoft.Recognizers.Text.DateTime/English/Extractors/EnglishTimeExtractorConfiguration.cs

```diff
@@ -97,6 +98,10 @@ public class EnglishTimeExtractorConfiguration : ITimeExtractorConfiguration
                 DateTimeDefinitions.ConnectNumRegex,
                 RegexOptions.IgnoreCase | RegexOptions.Singleline);
 
+        public static readonly Regex SpecialTimePattern =
+            new Regex(DateTimeDefinitions.SpecialTimePattern,
+                RegexOptions.IgnoreCase | RegexOptions.Singleline);
```

Comment:
Rename it to NumberEndingPattern, the same to all SpecialTimePattern.
This is not a good naming.